### PR TITLE
DOC: change Memorial Fund -> Technical Fellowship

### PR DESCRIPTION
**Pull request by tacaswell**
Minimal change from discussion in  #5558

### CODE REVIEW

File: doc/_templates/index.html [hunk 1]
```diff
@@ -89,8 +89,8 @@
       If you have benefited from John's many contributions, please say
       thanks in the way that would matter most to him.  Please consider
       making a donation to
-      the <a href="http://numfocus.org/johnhunter/">John Hunter Memorial
-      Fund</a>.</p>
+      the <a href="http://numfocus.org/johnhunter/">John Hunter Technical
```
Comment (efiring) [inline doc/_templates/index.html]: "Technical" -> "Technology"

Comment (tacaswell) [inline doc/_templates/index.html]: I swear I can read...

Comment (tacaswell) [inline doc/_templates/index.html]: fixed and force-pushed

**Comment by charles-cai** [comment]
https://scopatz.github.io/numfocus.org/johnhunter.html is it still valid?

**Comment by tacaswell** [comment]
I do not know, will look into it.

**Pull request merged**